.gitignore

```diff
@@ -1,2 +1,3 @@
 /target
 Cargo.lock
+/examples/frontend/build
```

Comment:
```suggestion
build
```

Reply:
Why?
It's the build directory created by the frontend example :thinking:

Reply:
All your build directories belong to us

Reply:
I mean, if you use `/` it is like an "absolute" path in the repository. If you don't use `/` then all directory called `build` will be ignored. No matter where they are in the tree.

In a project about tooling like that in Rust you need to exclude all directories called `target` and `build` because there will probably be more than one example.

Your CI should also run test that will build and run those crates to a certain extend.